include/deal.II/base/geometry_info.h

```diff
@@ -23,6 +23,338 @@
 
 DEAL_II_NAMESPACE_OPEN
 
+namespace internal
+{
+  template <int dim>
+  struct initialize_ucd_to_deal;
+
+  template <>
+  struct initialize_ucd_to_deal<1>
+  {
+    static constexpr std::array<unsigned int, 2>
+    value()
+    {
+      return {{0, 1}};
```

Comment:
Couldn't you avoid making this a function call that the compiler has to look through by just directly initializing a member variable?